src/main/kotlin/no/nav/medlemskap/regler/common/Personfakta.kt

```diff
@@ -107,7 +107,11 @@ class Personfakta(private val datagrunnlag: Datagrunnlag) {
     fun hentBrukerinputArbeidUtenforNorge(): Boolean = datagrunnlag.brukerinput.arbeidUtenforNorge
 
     fun hentBrukerLandskodeInnenfor12Mnd(): List<Adresse> {
-        return datagrunnlag.personhistorikk.bostedsadresser.filter { it.landkode == "NOR" && it.fom!!.isAfter(datohjelper.kontrollPeriodeForNorskAdresse().fom) }
+        return datagrunnlag.personhistorikk.bostedsadresser.filter {
+            it.landkode == "NOR"
+                    && (it.tom?.isAfter(datohjelper.kontrollPeriodeForNorskAdresse().fom) ?: it.tom == true)
```

Comment:
går det å erstatte it.tom == true med true ?

Reply:
Det var meningen å bare skrive true, blinksa litt der. Legger merke til at jeg ikke er helt våken ennå 😅